Cargo.toml

```diff
@@ -21,7 +21,7 @@ rand = "0.8"
 bitfield-rle = "0.2"
 parking_lot = "0.11"
 instant = "0.1"
-bytemuck = {version = "1.7", features = ["derive"]}
+bytemuck = {version = "1.9", features = ["derive"]}
```

Comment:
Pod trait was split in multiple traits in release 1.9.0 https://github.com/Lokathor/bytemuck/blob/main/changelog.md#190